cli/src/test/resources/fixture/cquery-test-android-code-change-android-impacted-targets.txt

```diff
@@ -1,2 +1,3 @@
-//src/main/java/com/integration:android
-//src/main/java/com/integration:guava-user
\ No newline at end of file
+@@//src/main/java/com/integration:android
+@@//src/main/java/com/integration:android_deployjars_internal_rule
```

Comment:
This is a bit odd, but it doesn't seem to be due to `--consistent_labels` change made in #244